jacodb-analysis/src/main/kotlin/org/jacodb/analysis/codegen/language/impl/JavaLanguage.kt

```diff
@@ -413,31 +442,17 @@ class JavaLanguage : TargetLanguage {
         }
     }
 
-    private fun appendStaticFunction(function: FunctionPresentation) = appendStartFunction(function.shortName, function)
-
     private fun classNameForStaticFunction(functionName: String): String {
-        return "ClassFor${functionName.capitalize()}"
-    }
-
-    private fun appendStartFunction(name: String, function: FunctionPresentation) {
-        writeSeparated("public class ${classNameForStaticFunction(name)}")
-        inScope {
-            tabulate()
-            writeVisibility(function.visibility)
-            writeSeparated("static")
-            writeTypeUsage(function.returnType)
-            write(function.shortName)
-            writeParametersList(function)
-            inScope {
-                appendLocalsAndSites(function)
-            }
+        val diff = Integer.valueOf(functionName.substring(11)) - Integer.MAX_VALUE / 2
```

Comment:
AFAIK there is constant in CodePresentation for Integer.MAX_VALUE / 2. Use it instead of magic numbers

Reply:
this constant is private. make it public, or just name the magic variable?